scripts/fetch-cloudwatch-metrics.py

```diff
@@ -0,0 +1,84 @@
+import boto3
+import sys
+import argparse
+from botocore.exceptions import ClientError
+from datetime import datetime, timedelta
+
+
+def generate_metric_query(hosted_zone_id, day_offset, region):
```

Comment:
Can see on [line 83](https://github.com/ministryofjustice/dns-iac/pull/370/files#diff-bd58e2658576b38ed89ac5f123bff0b64f88e53ce7b6b906c28fe7fab7a25d23R83) the days_offset is converted to an int which suggests typing might be important here.

Maybe add some type of annotations which will allow external tools (such as MyPy) to flag these issues in early development? i.e.
```python
def generate_metric_query(hosted_zone_id: str, day_offset: int, region: str) -> None:
```

Reply:
Although Python probably does magic under the hood to convert all of this on the fly, just my pickiness from a static typed background 😂

Reply:
Added! Along with some error handling for invalid region names